delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/ProjectsModule.scala

```diff
@@ -36,6 +41,25 @@ object ProjectsModule extends ModuleDef {
     ApiMappingsCollection(mappings)
   }
 
+  make[ProjectsCache].from { (config: AppConfig, as: ActorSystem[Nothing]) =>
+    ProjectsImpl.cache(config.projects)(as)
+  }
+
+  make[DeletionStatusCache].from { (config: AppConfig, as: ActorSystem[Nothing]) =>
+    ProjectsImpl.deletionCache(config.projects)(as)
+  }
+
+  make[ProjectsAggregate].fromEffect {
```

Comment:
We could make the different modules return the different tasks (delete files, caches, ...) to perform no ?

I am a little concerned about putting the aggregates at the distage level, it makes it easier to go around the module part and do stupid stuff if we don't pay attention

Reply:
This could be done as a future refactoring.